test/assignment.coffee

```diff
@@ -141,9 +141,6 @@ test "#1192: assignment starting with object literals", ->
 
 # Destructuring Assignment
 
-test "empty destructuring assignment", ->
-  {} = [] = undefined
-
```

Comment:
CS1 uses those for non-destructuring cases, though
```
coffee -bce> [v, []] = a
var v;

v = a[0], a[1];
```

Although it's... not very consistent...
```
coffee -bce> [[]] = a
a[0];
```